frontend/src/layers/styles/administrativeAndRegulatoryLayers.style.ts

```diff
@@ -239,33 +239,27 @@ const getLayerColor = (randomDigits, metadataIsShowed) => {
     case 0:
```

Comment:
Just a DRY note: why don't we make an array (or even an indexed object if needed) with the matching `theme.color` list and run `getStyle(getColorWithAlpha())` only once?

Reply:
+1
I added a Map to highlight the mapping